bliss/simulator/decoder.py

```diff
@@ -14,55 +14,49 @@
 class ImageDecoder(nn.Module):
     def __init__(
         self,
-        n_bands: int,
         pixel_scale: float,
-        psf_params_file: str,
         psf_slen: int,
-        sdss_bands: Tuple[int, ...],
+        sdss_fields: DictConfig,
     ) -> None:
         super().__init__()
-        assert n_bands == 1, "Only 1 band is supported"
 
-        self.n_bands = n_bands
+        self.n_bands = len(sdss_fields["bands"])
```

Comment:
Great. Yeah, there was no reason before to require a caller to specify `n_bands` when we already had list of the bands.